solc/CommandLineParser.cpp

```diff
@@ -718,7 +718,7 @@ General Information)").c_str(),
 		(CompilerOutputs::componentName(&CompilerOutputs::signatureHashes).c_str(), "Function signature hashes of the contracts.")
 		(CompilerOutputs::componentName(&CompilerOutputs::natspecUser).c_str(), "Natspec user documentation of all contracts.")
 		(CompilerOutputs::componentName(&CompilerOutputs::natspecDev).c_str(), "Natspec developer documentation of all contracts.")
-		(CompilerOutputs::componentName(&CompilerOutputs::metadata).c_str(), "Combined Metadata JSON whose Swarm hash is stored on-chain.")
+		(CompilerOutputs::componentName(&CompilerOutputs::metadata).c_str(), "Request the combined Metadata JSON whose Swarm hash is stored on-chain. If used with --output-dir the metadata will be written in that directory, one file per contract each named \"<ContractName>_meta.json\", otherwise it is displayed on stdout.")
```

Comment:
We should also change ``Swarm hash`` either to ``hash`` or ``IPFS hash`` or ``hash (see --metadata-hash)``...

Reply:
I see you changed it to `IPFS` now. Personally, I'd go with `hash` because otherwise we'll have to update it whenever we change the default.